.travis.yml

```diff
@@ -20,10 +19,11 @@ addons:
    postgresql: "9.4"
 
 before_script:
-   - psql -c "CREATE ROLE full_access;" -U postgres
+   # Used by (so far non-existent) spec tests for xmlforms
```

Comment:
I see references here to a postgresql database used for couch2pg integration tests:
https://github.com/medic/medic-analytics/blob/2124.reflect-edited-and-deleted-rebased/spec/couch2pg/index.js#L8-L12

How does that database get created in Travis?